test/functional/JIT_Test/src/jit/test/finalizer/TestObject.java

```diff
@@ -66,12 +66,10 @@ public static boolean isFinalized(String name) {
 	}
 	
 	protected static void thoroughGCandFinalization() {
-		System.gc();
 		System.gc();
 		System.runFinalization();
```

Comment:
@dmitripivkine Do we still need to call runFinaliization() and do the additional System.g()? According to our doc update the finalization should occur from the single System.gc() now.

Reply:
> @dmitripivkine Do we still need to call runFinaliization() and do the additional System.gc()? According to our doc update the finalization should occur from the single System.gc() now.

In theory your statement is correct. However I don't know the reason why we call `System.gc()` and `System.runFinalization()` multiple times. I guess there is a reason. We can try to simplify this code (calling System.gc() twice?) and watch for intermittent failures.